prisma/schema.prisma

```diff
@@ -267,8 +283,19 @@ model GeneratedK1 {
 // Hosted Image
 // -----------------
 model HostedImage {
-  id        String   @id
-  filename  String
-  createdAt DateTime @default(now())
-  is_used   Boolean  @default(false)
+  id                Int      @id @default(autoincrement())
+  filename          String
+  provider_image_id String
+  provider          String
+  url               String
+  createdAt         DateTime @default(now())
+  is_used           Boolean  @default(false)
+
+  ProjectThumbnail    Project[]   @relation("ProjectThumbnail")
```

Comment:
Is this here a many-to-many relationship???
Does a single hosted image have multiple projectThumbnail accossiated with it??

Reply:
No it's not, but Prisma doesn't let me remove these lines, especially for the Project since there is multiple reference to HostedImage. Prisma need named Relation

Reply:
Yup, prisma requires both side to state that the have a relation to each other.
I think it's possible to do:
```js
model Project {
  id                 Int      @id @default(autoincrement())
  ...
  thumbnail_image_id  String? @unique
  thumbnail_image_rel HostedImage? @relation( fields: [thumbnail_image_id], references: [id])
 ```
 
 then
 ```js
 model HostedImage {
  id        String   @id
  filename  String
  createdAt DateTime @default(now())
  is_used   Boolean  @default(false)

  project_thumbnail Project?
}
```

Reply:
Gotcha, I'll try later today. I'll let you know